corelib/src/superpoint_rpautrat/SuperpointRpautrat.cpp

```diff
@@ -0,0 +1,247 @@
+/**
+ * SuperPoint implementation based on the PyTorch version by Rémi Pautrat, Paul-Edouard Sarlin
+ * Adapted for RTAB-Map integration
+ */
+
+#include "SuperpointRpautrat.h"
+#include <rtabmap/utilite/ULogger.h>
+#include <rtabmap/utilite/UDirectory.h>
+#include <rtabmap/utilite/UFile.h>
+#include <pybind11/embed.h>
+#include <torch/torch.h>
+#include <torch/script.h>
+#include <opencv2/opencv.hpp>
+
+namespace rtabmap
+{
+
+// Run the python script to export the SuperPoint model file with the desired parameters
+static std::string exportSuperPointTorchScript(
+    const std::string & superpointDir,
+    const int & width,
+    const int & height,
+    const float & threshold,
+    const int & nms_radius,
+    const bool & cuda)
+{
+    // Resolve paths
+    const std::string srcDir = UDirectory::getDir(__FILE__); // folder of this .cpp at build time
```

Comment:
That won't work when rtabmap is installed and/or deployed. I think the best would be to include `superpoint_to_torchscript.py` as a "resource" in rtabmap library. That can be done by appending `superpoint_to_torchscript.py` to RESOURCES here https://github.com/introlab/rtabmap/blob/a3076392b5fc4273a6d42f03261bfc6281c02a58/corelib/src/CMakeLists.txt#L829 (only append if we are building with Torch and python).

The way it works afterwards is that you can include the resource file and decode it into a string:
```cpp
#include "superpoint_to_torchscript_py.h"
[...]
std::string file_content = uHex2Str(SUPERPOINT_TO_TORSCRIPT_PY);
```

Reply:
follow-up comment: https://github.com/introlab/rtabmap/pull/1603/files#r2487984815